Include/internal/pycore_lock.h

```diff
@@ -92,6 +92,13 @@ PyMutex_IsLocked(PyMutex *m)
     return (_Py_atomic_load_uint8(&m->v) & _Py_LOCKED) != 0;
 }
 
+// Re-initializes the mutex after a fork to the unlocked state.
+static inline void
+_PyMutex_at_fork_reinit(PyMutex *m)
+{
+    memset(m, 0, sizeof(*m));
```

Comment:
This is the same as `*m = (PyMutex}{0};`, right?  I'm fine with using `memset()`, though I find the initializer version a little easier to follow.  I'm not aware of the tradeoffs, so I'll leave it up to you.

Reply:
I originally wrote `*m = (PyMutex}{0};`, but that broke the Windows build. `*m = (PyMutex}{0};` is valid C,  but it is not valid C++. For some reason the Windows build compiles this header as part of a C++ file.

Reply:
blech!  okay then